Algorithm/QCAlgorithm.Framework.cs

```diff
@@ -400,18 +411,7 @@ public void EmitInsights(params Insight[] insights)
         /// <param name="insight">The insight to be emitted</param>
         public void EmitInsights(Insight insight)
         {
-            if (_autogeneratedOrderBasedInsightWasCalled)
-            {
-                throw new InvalidOperationException("EmitInsights should be called before placing an order." +
-                    " Algorithms typically follow the classic design of simply placing trades, or they should" +
-                    " follow our recommendation of emitting insights and letting the execution models handle" +
-                    " the trading. To distinguish between these styles we monitor which comes first, and insight" +
-                    " or a trade.");
```

Comment:
Is this handled somewhere else?

Reply:
Yes, it was duplicated in the other `EmitInsights` implementation (above) which now we will call